integrations/lambda/demo.ipynb

```diff
@@ -281,6 +281,148 @@
     "\n",
     "        letters = \"ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789\"\n",
     "        return ''.join(random.choice(letters) for _ in range(length))\n",
+    "    \n",
+    "    def generate_dashboard(self, grafana_data_source_name: str, database_name: str, table_name: str) -> dict:\n",
+    "        # Each measure will have a panel\n",
+    "        panels = []\n",
+    "        for i, measure_template in enumerate(self.measure_templates):\n",
+    "            measure_name = measure_template['name']\n",
+    "            query = \"\"\n",
+    "            if len(self.measure_templates) > 1:\n",
+    "                query = f\"SELECT time, \\\"{measure_name}\\\", {', '.join(f\"\\\"{dimension_template['name']}\\\"\" for dimension_template in self.dimension_templates)} FROM \\\"{database_name}\\\".\\\"{table_name}\\\" ORDER BY time ASC\"\n",
```

Comment:
I get the following error when using Amazon Sagemaker and running locally. In general it would be best to avoid nested F-strings if possible.

```
    query = f"SELECT time, \"{measure_name}\", {', '.join(f"\"{dimension_template['name']}\"" for dimension_template in self.dimension_templates)} FROM \"{database_name}\".\"{table_name}\" ORDER BY time ASC"
                                                             ^
SyntaxError: unexpected character after line continuation character
```

Reply:
Strange, I did not get this error. I have removed the nested F-string, let me know if you are still getting this error.

Reply:
No idea why you were not seeing the error. After your fix I was able to run the playbook.